common/src/Renderer/BrushRenderer.h

```diff
@@ -164,25 +156,25 @@ namespace TrenchBroom {
             Color m_tintColor;
             bool m_showOccludedEdges;
             Color m_occludedEdgeColor;
+            bool m_transparent;
             float m_transparencyAlpha;
             
             bool m_showHiddenBrushes;
         public:
             template <typename FilterT>
             explicit BrushRenderer(const FilterT& filter) :
-            m_filter(new FilterT(filter)),
+            m_filter(std::make_unique<FilterT>(filter)),
             m_showEdges(false),
             m_grayscale(false),
             m_tint(false),
             m_showOccludedEdges(false),
+            m_transparent(false),
             m_transparencyAlpha(1.0f),
             m_showHiddenBrushes(false) {
                 clear();
             }
             
-            explicit BrushRenderer(bool transparent);
```

Comment:
I've already forgotten what the meaning of this constructor was :(
Did `BrushRenderer(false)` create a renderer that only rendered the opaque faces?

Reply:
I think passing true forced transparent rendering. This was necessary for the clip tool where the discarded parts of the clipped brushes would be rendered transparent.